python/cudf/cudf/utils/metadata/orc_column_statistics.proto

```diff
@@ -0,0 +1,61 @@
+message IntegerStatistics  {
+  optional sint64 minimum = 1;
+  optional sint64 maximum = 2;
+  optional sint64 sum = 3;
+}
+
+message DoubleStatistics {
+  optional double minimum = 1;
+  optional double maximum = 2;
+  optional double sum = 3;
+}
+
+message StringStatistics {
+  optional string minimum = 1;
+  optional string maximum = 2;
+  // sum will store the total length of all strings in a stripe
+  optional sint64 sum = 3;
+}
+
+message BucketStatistics {
+  repeated uint64 count = 1 [packed=true];
+}
+
+message DecimalStatistics {
+  optional string minimum = 1;
+  optional string maximum = 2;
+  optional string sum = 3;
+}
+
+message DateStatistics {
+  // min,max values saved as days since epoch
+  optional sint32 minimum = 1;
+  optional sint32 maximum = 2;
+}
+
+message TimestampStatistics {
+  // min,max values saved as milliseconds since epoch
+  optional sint64 minimum = 1;
+  optional sint64 maximum = 2;
+  // min,max values saved as milliseconds since UNIX epoch
+  optional sint64 minimumUtc = 3;
+  optional sint64 maximumUtc = 4;
```

Comment:
What are the difference between these two? Aren't epoch and UNIX epoch the same?

Reply:
`minimum`/`maximum` are localized by timezone of the file writer while `minimumUtc`/`maximumUtc` are adjusted to UTC. `minimum`/`maximum` are legacy fields in ORC format that aren't currently handled as described in an above TODO [1].

I removed the second comment line though so it isn't confusing as they both are measured from epoch but one is localized to timezone of writer.

[1] https://github.com/rapidsai/cudf/pull/6142/files/bac0c8f964c2d98c1b32cfbba6be45f4d4edc587#diff-7ff4f0a7358704f99e4fda4c3b108d06R58-R62